rmw_connextdds_common/src/common/rmw_context.cpp

```diff
@@ -91,35 +91,15 @@ rmw_connextdds_initialize_participant_qos(
     return RMW_RET_ERROR;
   }
 
-  /* Lookup and configure initial peer from environment */
-  const char * initial_peers = nullptr;
-  const char * lookup_rc =
-    rcutils_get_env(RMW_CONNEXT_ENV_INITIAL_PEERS, &initial_peers);
-
-  if (nullptr != lookup_rc || nullptr == initial_peers) {
-    RMW_CONNEXT_LOG_ERROR_A_SET(
-      "failed to lookup from environment: "
-      "var=%s, "
-      "rc=%s ",
-      RMW_CONNEXT_ENV_INITIAL_PEERS,
-      lookup_rc)
-    return RMW_RET_ERROR;
-  }
-
-  if ('\0' != initial_peers[0]) {
-    rmw_ret_t rc = rmw_connextdds_parse_string_list(
-      initial_peers,
-      &dp_qos.discovery.initial_peers,
-      ',' /* delimiter */,
-      true /* trim_elements */,
-      false /* allow_empty_elements */,
-      false /* append_values */);
-    if (RMW_RET_OK != rc) {
-      RMW_CONNEXT_LOG_ERROR_A(
-        "failed to parse initial peers: '%s'", initial_peers)
-      return rc;
+  if (ctx->participant_qos_override_policy ==
+    rmw_context_impl_t::participant_qos_override_policy_t::All &&
+    DDS_StringSeq_get_length(&ctx->initial_peers) > 0)
+  {
+    printf("custom peers!\n");
```

Comment:
I guess this is a leftover from debugging

```suggestion
```

Reply:
Oops, fixed in 4e4fd65